app/models/token_sender.rb

```diff
@@ -1,7 +1,7 @@
 class TokenSender
   def initialize(user, options={})
     @user = user
-    @client = options[:client] || Twilio::REST::Client.new(ENV["twilio_key"], ENV["twilio_s_id"])
+    @client = options[:client] || Twilio::REST::Client.new(ENV["twilio_s_id"], ENV["twilio_key"])
   end
```

Comment:
Twilio requires these keys to be in the new order. I realized that my tests had the keys in the right order, but this PORO didn't, so it broke when I tried to change my password in the browser.